### DISCUSSION TITLE
Extension uses some wrong API parameter names

### DISCUSSION THREAD

**Discussion by wiebeck** [quarkiverse/quarkus-vault#325] [General]
The extension uses some API parameters with wrong letter case and wrong hyphens for `sys/health` and `sys/init` endpoints.

This PR fixes that: #320 

I hope this PR is enough - do you need anything else?

**Comment by kdubb** [comment]
Just a case of maintainers having a lot of other stuff to do. I'll check out your PR soon. Thanks for contributing (and apparently sleuthing through the code to figure out the custom generator format I created haha)
